src/components/RiskComponentDropdown.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <v-expansion-panel class="riskComponentDropdown">
+    <v-expansion-panel-header class="d-flex flex-row">
+      <div :class="riskClass" class="componentIconContainer" flex-shrink-1>
+        <CrowdingIcon v-if="isCrowding" class="componentIcon" />
+        <DropletsIcon v-if="isDroplets" class="componentIcon" />
+        <TimeIcon v-if="isTime" class="componentIcon" />
+        <VentIcon v-if="isVent" class="componentIcon" />
+      </div>
+      <div class="componentHeaderText" flex-grow-1>
+        <div class="componentTitle">{{ title }}</div>
+        <div class="componentRiskLabel">{{ risk }}</div>
+      </div>
+    </v-expansion-panel-header>
+    <v-expansion-panel-content>
+      <Markdown :source="notes" />
+    </v-expansion-panel-content>
+  </v-expansion-panel>
+</template>
+
+<script>
+import { mapGetters } from "vuex";
+import CrowdingIcon from "@/assets/Risk_Crowding-filled.svg";
+import DropletsIcon from "@/assets/Risk_Droplets-filled.svg";
+import TimeIcon from "@/assets/Risk_Time-filled.svg";
+import VentIcon from "@/assets/Risk_Ventilation-filled.svg";
+import Markdown from "vue-markdown";
+
+export default {
+  components: {
+    CrowdingIcon,
+    DropletsIcon,
+    TimeIcon,
+    VentIcon,
+    Markdown
+  },
+  props: {
+    activity: Object,
+    type: String
+  },
+  computed: {
+    ...mapGetters(["components"]),
+    isCrowding() {
+      return this.type === "crowding";
+    },
+    isDroplets() {
+      return this.type === "droplets";
+    },
+    isTime() {
+      return this.type === "exposureTime";
+    },
+    isVent() {
+      return this.type === "ventilation";
+    },
+    title() {
+      return this.components[this.type].title;
+    },
+    score() {
+      return parseInt(this.activity[this.type]);
+    },
+    risk() {
+      return this.score === 1 ? "Low" : this.score === 2 ? "Medium" : "High";
+    },
+    notes() {
+      return this.type !== "" ? this.activity[this.type + "Notes"] : "";
```

Comment:
empty strings are falsy, so you can just check if `this.type` is truthy
```suggestion
      return this.type ? this.activity[`${this.type}Notes`] : "";
```

Reply:
Fixed